File: login-server/src/main/java/work/fking/pangya/login/packet/outbound/ConfirmCharacterReplies.java
package work.fking.pangya.login.packet.outbound;

import work.fking.pangya.networking.protocol.OutboundPacket;

public final class ConfirmCharacterReplies {

    private static final int PACKET_ID = 0x11;

    private ConfirmCharacterReplies() {
    }

    public static OutboundPacket ok() {
        return buffer -> {
            buffer.writeShortLE(PACKET_ID);
            buffer.writeByte(0);
        };
    }
}

File: login-server/src/main/kotlin/work/fking/pangya/login/packet/outbound/ConfirmCharacterReplies.kt
package work.fking.pangya.login.packet.outbound

import work.fking.pangya.networking.protocol.OutboundPacket

object ConfirmCharacterReplies {

    fun ok(): OutboundPacket {
        return OutboundPacket { buffer ->
            buffer.writeShortLE(0x11)
            buffer.writeByte(0)
        }
    }
}
